ls1intum/Artemis
# `Lectures`: Make the connection between learning objectives clearer

### Checklist
#### General
#### Client

### Motivation and Context
In a previous PR I added a wizard mode for editing/creating lectures. There it's become possible to directly create/edit learning goals and connect them to units. However, in several tests it's become obvious that many users cannot see the connecting table at the bottom of the form and were not sure how to connect goals and units.

### Description
In order to fix this, the learning goal step now shows another option for each goal "Connect", which will only open the connecting table so it's clearer and more obvious for the users where to connect goal and unit.

### Steps for Testing
Prerequisites:
- 1 Instructor who can create lectures

1. Log in to Artemis
2. Navigate to Course Management
3. Edit a lecture
4. Go to the learning goals step
5. (If no goal exists, create one)
6. The learning goal in the list should have the new connect option
7. Clicking that opens the table for connecting the units

### Review Progress

#### Code Review
#### Manual Tests

### Screenshots
![Bildschirm­foto 2023-01-26 um 02 15 25](https://user-images.githubusercontent.com/16115423/214738741-bad524aa-cd32-40e7-938e-68f80719b69a.png)
![Bildschirm­foto 2023-01-26 um 02 15 32](https://user-images.githubusercontent.com/16115423/214738733-a63c5936-1c04-4ae1-ab5a-9e2da9bb2998.png)
